stock_available_to_promise_release/models/stock_move.py

```diff
@@ -277,7 +346,16 @@ def _should_compute_ordered_available_to_promise(self):
         )
 
     def _action_cancel(self):
+        # Check that it is a released move...
+        # If yes unreleased it
+        moves_to_unrelease = self.filtered(lambda m: m.unrelease_allowed)
+        moves_to_unrelease.unrelease()
+        # moves_origin = self.move_orig_ids
+        # __import__("pdb").set_trace()
```

Comment:
remove commented pdb

Reply:
Fix on https://github.com/OCA/wms/pull/556, will be be visible here after rebase.